StatBot/Models/DataModel.cs

```diff
@@ -16,6 +16,7 @@ public static List<User> Users
             get
             {
                 var db = new PostgresUserRepository();
+                
                 return db.GetAll().ToList();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor 'Users' into a method, properties should not copy collections. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS2365)